_src/static/sass/base/_variables.scss

```diff
@@ -0,0 +1,156 @@
+/* ============================
+-------------------------------
+ 1. Grid
+ 2. Fonts
+ 3. Colors (Palette)
+ 4. Colors (Application)
+ 5. Timing
+-------------------------------
+============================ */
+
+$baseline:     20px;
```

Comment:
Let's use rems for a unit of measurement here. Bourbon's Pixel to Rems utility (http://bourbon.io/docs/#px-to-rem) is a godsend for this. the syntax would then be ``rem(20)``.